plugins/provenance/server/resource.py

```diff
@@ -323,8 +325,10 @@ def snapshotResource(self, obj):
         return snap
 
     def fileSaveHandler(self, event):
-        """When a file is saved, update the provenance of the parent item.
-        :param event: the event with the file information."""
+        """
+        When a file is saved, update the provenance of the parent item.
+        :param event: the event with the file information.
```

Comment:
I think it would be great to run sphinx as a test and look for errors. It's fine to do that on a different branch.